tests/integration/individual_response/test_individual_response.py

```diff
@@ -0,0 +1,375 @@
+        # When I use an invalid id in an individual response url
+        self.get("/")
+        self.get(f"individual-response/not-an-id/how")
+        self.assertStatusCode(404)
+
+    def test_how_route_redirects_signed_out(self):
+        # Given the hub is enabled
+        # And I add a housheold member
+        self._add_household_no_primary()
+        self.post()
+
+        # When I sign out and navigate to the individual response page
+        individual_response_link = self.individual_response_link
+        self.get(individual_response_link)
+        self.post()
+        self.get(self.sign_out_link)
+        self.get(individual_response_link)
+
+        # Then I should see the signed out page
+        self.assertInBody("Your session has expired")
+
+    def test_404_when_hub_not_accessible(self):
+        # Given the hub is not enabled
+        # And I try to navigate to the individual response page
+        self.get("/individual-response")
+
+        # Then I should be redirected
+        self.assertInBody("Page not found")
+
+    def test_how_404_when_hub_not_accessible(self):
+        # Given the hub is not enabled
+        # And I try to navigate to the individual response how page
+        self.get("/individual-response/fake-id/how")
+
+        # Then I should be redirected
+        self.assertInBody("Page not found")
+
+    def test_confirm_404_when_hub_not_accessible(self):
+        # Given the hub is not enabled
+        # And I try to navigate to the individual response how page
+        self.get("/individual-response/fake-id/post/confirm-address")
+
+        # Then I should be redirected
+        self.assertInBody("Page not found")
+
+    def test_post_confirmation_404_when_hub_not_accessible(self):
+        # Given the hub is not enabled
+        # And I try to navigate to the individual response how page
+        self.get("/individual-response/post/confirmation")
+
+        # Then I should be redirected
+        self.assertInBody("Page not found")
+
+    def test_ir_guidance_not_displayed_when_no_household_members(self):
+        # Given the hub is enabled
+        # And I navigate to the hub after adding a primary person
+        self._add_no_household_members()
+
+        # Then I should not see the individual response guidance
+        self.assertInBody("Submit survey")
+        self.assertNotInBody("If you can't answer someone else's questions")
+        self.assertNotInBody("Hide")
+
+    def test_ir_guidance_not_displayed_when_only_primary(self):
+        # Given the hub is enabled
+        # And I navigate to the hub after adding a primary person
+        self._add_primary()
+
+        # Then I should not see the individual response guidance
+        self.assertInBody("Choose another section to complete")
+        self.assertNotInBody("If you can't answer someone else's questions")
+        self.assertNotInBody("Hide")
+
+    def test_ir_guidance_not_displayed_when_primary(self):
```

Comment:
As the guidance can appear on the hub or individual section, the naming needs to be clearer. 
```suggestion
    def test_individual_section_ir_guidance_not_displayed_when_primary(self):
```

This applies to all of the tests for testing the individual section guidance.

Alternatively, it could be worth considering splitting the tests up into hub, individual section and individual response tests.